### PR TITLE
gn: fix hooks

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **gn/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/gn/all/conandata.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 sources:
   "cci.20210429":
     url: "https://gn.googlesource.com/gn/+archive/6771ce569fb4803dad7a427aa2e2c23e960b917e.tar.gz"
+    sha256: "d05eb257ebf9aeaf9238346624f2a30e91e1e7a0ca78eb0e0e42274591338443"
```
Comment (madebr) [inline recipes/gn/all/conandata.yml:4]: This hash is not stable, I just downloaded the archive and got `80bcb40e082489588b9572082116ce2c1f18a0a7f89c7d63173e250002a57ccf` now.

Comment (madebr) [reply recipes/gn/all/conandata.yml:4]: Perhaps add a comment about it not being stable to avoid future attempts re-adding it.

Comment (ericLemanissier) [reply recipes/gn/all/conandata.yml:4]: Ok, so we need to add it as exception to the hook

Comment (ericLemanissier) [reply recipes/gn/all/conandata.yml:4]: Ok, so we need to add it as exception to the hook

**Pull request closed without merging**